with-lib/mutual-tls-example/certs/CLIENT_ID.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCj0soAkxb8dQ3d
```

Comment:
Does this belong to a real production client? If so, should be revoke this cert given that it's exposed?